common/src/main/java/coffee/waffle/emcutils/Caches.java

```diff
@@ -0,0 +1,33 @@
+package coffee.waffle.emcutils;
+
+import coffee.waffle.emcutils.feature.Nameplates;
+import com.google.common.cache.*;
```

Comment:
Would prefer no star imports but idrc that much

Reply:
Yeah... I prefer no star imports too... I must've lost my IntelliJ configs :(